### PR TITLE
Removed precedence from parameter definitions in dashboard

### PR DESCRIPTION
**Pull request by jlstevens**
This PR removes the `precedence` settings of the parameters used in the dashboard. Once https://github.com/pyviz/panel/pull/238 is merged, this definition order will automatically be respected (at least for Python 3.6+ - earlier versions will be sorted alphanumerically).

### CODE REVIEW

**Comment by jlstevens** [comment]
@jbednar Would this be a good reason to pin panel to the most recent version (assuming that panel PR is merged and is released?)

**Comment by jbednar** [comment]
We'll probably need to pin for the other upcoming changes to param and panel, but for now it's not crucial.

**Comment by jlstevens** [comment]
Just to note that https://github.com/pyviz/panel/pull/238 has been merged....

File: examples/dashboard.ipynb [hunk 1]
```diff
@@ -81,18 +81,18 @@
...
+    "    normalization = pm.ObjectSelector( default='eq_hist',       objects=norms)\n",
+    "    cmap          = pm.ObjectSelector( default=cmaps['fire'],   objects=cmaps)\n",
+    "    spreading     = pm.Integer(        default=0,               bounds=(0, 5))\n",
     "    \n",
-    "    basemap       = pm.ObjectSelector( precedence=0.18, default=bases['EsriImagery'], objects=bases)\n",
-    "    data_opacity  = pm.Magnitude(      precedence=0.20, default=1.00, doc=\"Alpha value for the data\")\n",
-    "    map_opacity   = pm.Magnitude(      precedence=0.22, default=0.75, doc=\"Alpha value for the map\")\n",
-    "    show_labels   = pm.Boolean(        precedence=0.24, default=True)\n",
+    "    basemap       = pm.ObjectSelector( default=bases['EsriImagery'], objects=bases)\n",
+    "    data_opacity  = pm.Magnitude(      default=1.00, doc=\"Alpha value for the data\")\n",
```
Comment (jbednar) [inline examples/dashboard.ipynb]:
```suggestion
    "    data_opacity  = pm.Magnitude(1.00)\n",
```

File: examples/dashboard.ipynb [hunk 2]
```diff
@@ -81,18 +81,18 @@
...
+    "    map_opacity   = pm.Magnitude(      default=0.75, doc=\"Alpha value for the map\")\n",
```
Comment (jbednar) [inline examples/dashboard.ipynb]:
```suggestion
    "    map_opacity   = pm.Magnitude(0.75)\n",
```

File: examples/dashboard.ipynb [hunk 3]
```diff
@@ -81,18 +81,18 @@
...
+    "    show_labels   = pm.Boolean(        default=True)\n",
```
Comment (jbednar) [inline examples/dashboard.ipynb]:
```suggestion
    "    show_labels   = pm.Boolean(True)\n",
```

File: examples/dashboard.ipynb [hunk 4]
```diff
@@ -81,18 +81,18 @@
...
+    "    plot          = pm.ObjectSelector( default=source.plots[0], objects=plots)\n",
+    "    field         = pm.ObjectSelector( default=field,           objects=fields)\n",
+    "    agg_fn        = pm.ObjectSelector( default=ds.count,        objects=aggfns)\n",
     "    \n",
-    "    normalization = pm.ObjectSelector( precedence=0.13, default='eq_hist',       objects=norms)\n",
-    "    cmap          = pm.ObjectSelector( precedence=0.14, default=cmaps['fire'],   objects=cmaps)\n",
-    "    spreading     = pm.Integer(        precedence=0.16, default=0,               bounds=(0, 5))\n",
+    "    normalization = pm.ObjectSelector( default='eq_hist',       objects=norms)\n",
+    "    cmap          = pm.ObjectSelector( default=cmaps['fire'],   objects=cmaps)\n",
+    "    spreading     = pm.Integer(        default=0,               bounds=(0, 5))\n",
```
Comment (jbednar) [inline examples/dashboard.ipynb]:
```suggestion
    "    spreading     = pm.Integer(0, bounds=(0, 5))\n",
```

**Pull request merged**